react-app/src/providers/WalletProvider.tsx

```diff
@@ -152,6 +153,13 @@ const WalletProvider: React.FC<WalletProviderProps> = (props) => {
     }
   }, [activeWallet, autoConnectWalletType, connectToKeplr, walletStatus]);
 
+  // https://docs.keplr.app/api/#change-key-store-event
```

Comment:
do we need to handle network change as well? (required for metamask, not sure about keplr)

Reply:
doesn't seem to be documented, also doesn't seem like it would affect transactions when in a different network as tested